requirements.txt

```diff
@@ -6,3 +6,4 @@ sympy
 numpy
 find_libpython
 scikit-build
+importlib_resources;python_version<"3.9"
```

Comment:
Has something, e.g. CI, or someone, e.g. you, run this with Python 3.8?

Reply:
The CI builds it on 3.8 in multiple places, for instance:
https://github.com/BlueBrain/nmodl/blob/c0eb1ac07bcfcb80189170819c52e4d7645f7c72/azure-pipelines.yml#L205
https://github.com/BlueBrain/nmodl/blob/c0eb1ac07bcfcb80189170819c52e4d7645f7c72/.github/workflows/nmodl-ci.yml#L19

Reply:
Perfect!